File: chapter1/loops/mario/mario3.c
// Stacks blocks (#) on top of eachother
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get the height from the user
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 0);

    // Print the blocks stacked
    for (int i = 0; i < height; i++)
    {
        printf("#\n");
    }
}

File: chapter6/python_more/mario3.py
from cs50 import get_int

def main():
    # Get a positive integer from the user
    n = get_positive_int("Height: ")

    # Print out a square set of blocks
    for i in range(n):
        for j in range(n):
            print("#", end="")
        print()

def get_positive_int(s):
    while True:
        n = get_int(s)
        if n > 0:
            break
    return n
    
main()    
